### ISSUE TITLE
How to make LR scheduler verbose?

### ISSUE THREAD

**Issue by Molaire** [PyTorchLightning/pytorch-lightning#2390]
### Before asking:   
1. search the issues.   
2. search the docs.    

#### What is your question?
Hi, I am currently using the ReduceLROnPlateau scheduler and returning it as a [dictionnary] in the configure_optimizers method. All other options seem to work, but I cannot seem to be able to make the scheduler verbose. 
Am I missing something?
#### Code
    def configure_optimizers(self):
        optim = torch.optim.Adam(self.parameters(), lr=self.learning_rate)
        sched = {
            "scheduler": torch.optim.lr_scheduler.ReduceLROnPlateau(optim),
            "verbose": True,
            "monitor": 'val_loss',
            "mode": "min",
            "patience": self.hparams.scheduler_patience,
            "interval": 'epoch',
            "frequency": 1,
            "eps": 1E-6
        }
        return [optim], [sched]

#### What's your environment?

 - OS: Win10
 - Packaging pip
 - Version pl 0.8.1 torch 1.6

**Comment by rohitgr7** [comment]
Some of these values are scheduler specific: https://pytorch.org/docs/stable/optim.html#torch.optim.lr_scheduler.ReduceLROnPlateau

While others are lightning specific: 
```
sched = {
         'scheduler': ReduceLROnPlateau(optimizers[0], ...),
         'monitor': 'val_recall', # Default: val_loss
         'interval': 'epoch',
         'frequency': 1
      }
```
